live-site/components/event-schedule-section/event-item-tag/EventItemTag.styles.ts

```diff
@@ -0,0 +1,99 @@
+import styled from 'styled-components';
+import { max } from '../../../../shared-ui/lib/responsive';
+import { P } from '../../../../shared-ui/style/typography';
+import { StyledTagAndTagTextProps } from '../../../lib/types';
+
+const StyledTag = styled.div<StyledTagAndTagTextProps>`
+  background-color: white;
+  border-color: ${(StyledTagAndTagTextProps): string => {
+    switch (StyledTagAndTagTextProps.type) {
+      case 'Meal':
+        return 'firebrick';
```

Comment:
for all of these colors can we put them in the colors object? and do colors.PURPLE or something especially since they're used multiple times here